services/admin-panels/docker-compose.yml.j2

```diff
@@ -97,7 +97,11 @@ services:
           - node.role == manager
       resources:
         limits:
-          memory: 2048M
+          memory: "2G"
+          cpus: "0.1"
```

Comment:
you will not be able to run anything in decent time with such low CPU limit

Reply:
raised